wranglertools/import_data.py

```diff
@@ -793,9 +793,7 @@ def error_report(error_dic, sheet, all_aliases, connection, error_id=''):
                 nf_txt = 'not found'
                 not_found = None
                 alias_bit = None
-                if error_id:
```

Comment:
Yeah, this was the bit that I saw was causing the issue - removing this should do the trick.